featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -116,7 +166,17 @@ def get_function(self):
 
 
 class Max(AggregationPrimitive):
-    """Finds the maximum non-null value of a numeric feature."""
+    """Finds the maximum value of a numeric feature.
```

Comment:
this one line should be updated to follow the standards of other primitives

Reply:
Fixed.